### ISSUE TITLE
Beginner Needs help please!

### ISSUE THREAD

**Issue by KreatorCellAya** [gepd/Deviot#344]
First off all, I am not a programmer... which might be a good thing because I am the worst with all this programming installation.  I have spent this entire day trying to get this working... probably about 18 hours straight at this point, here's where I am...

I've coded in Sublime in the past, so that's what I'm using.  However I just today (yesterday) installed it on my system (in the past I just ran it from a drive).  I also updated to the newest 4180 build.  Installed package manager, Deviot... text in the status bar says it installed, but I don't get a menu item like in the pictures (even after restarts).

I finally found how to install Python... I'm having to run Python 3.8.10 because that's the newest version that operates on Windows 7 64 bit.

I've found and installed PlatformIO core CLI (I believe that worked, I had to run the script in Sublime, but it seemed to download and install files, however I have no way of verifying that)

I cannot find the PIO debug files piodebuggdb anywhere. 

I'm hoping that's why I don't have Deviot in Sublime.  But I'm so lost with all of this, and extremely frustrating.  There's no clear information or instructions anywhere, and things aren't installing correctly. 

Please help me out!

Thank you so much for your time.

**Comment by KreatorCellAya** [comment]
Also... I have downloaded the Deviot Develop files from get hub (I don't really understand that website either, but I believe I've learnt how to download files from there - again, they don't make that easy).

I just don't know what to do with all those files.  I've looked through every folder, no exe.  I've run various Python files (through Sublime)... the best I got was running one file I got something like "Sublime doesn't exist"

Which is perplexing, because it's literally running inside Sublime, AND (contrary to my usual installs) Sublime is in its default directory and default installation.  So I don't know why none of this stuff finds each other, and nothings talking to each other.

**Comment by KreatorCellAya** [comment]
Can no one help me install this?

**Comment by KreatorCellAya** [comment]
Okay.... so I fixed it and it's working now!

Thankfully this resource helped: https://github.com/gepd/Deviot/issues/337

I just had to copy the file into the proper directory.  First I downloaded his file and placed it as indicated, at which point I got the menu item in Sublime.  But nothing really worked, everything was grayed out, so I decided to copy my Deviot folder files into that same folder, restarted Sublime, and pow... now I have something that looks like it works. 

Thank you so much!!

Hopefully others can use this fix as well!
